
From: Ingo Molnar <mingo@elte.hu>

the patch below removes stale references to kernel/hardirq.c in comments,
remnants of the earlier iterations of the generic irq subsystem code.

Signed-off-by: Ingo Molnar <mingo@elte.hu>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/i386/Kconfig        |    2 +-
 25-akpm/arch/ppc64/Kconfig       |    2 +-
 25-akpm/arch/x86_64/Kconfig      |    2 +-
 25-akpm/arch/x86_64/kernel/irq.c |    2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff -puN arch/i386/Kconfig~doc-remove-references-to-hardirqc arch/i386/Kconfig
--- 25/arch/i386/Kconfig~doc-remove-references-to-hardirqc	2004-10-03 16:42:22.186080144 -0700
+++ 25-akpm/arch/i386/Kconfig	2004-10-03 16:42:22.194078928 -0700
@@ -1195,7 +1195,7 @@ source "crypto/Kconfig"
 source "lib/Kconfig"
 
 #
-# Use the generic interrupt handling code in kernel/hardirq.c:
+# Use the generic interrupt handling code in kernel/irq/:
 #
 config GENERIC_HARDIRQS
 	bool
diff -puN arch/ppc64/Kconfig~doc-remove-references-to-hardirqc arch/ppc64/Kconfig
--- 25/arch/ppc64/Kconfig~doc-remove-references-to-hardirqc	2004-10-03 16:42:22.187079992 -0700
+++ 25-akpm/arch/ppc64/Kconfig	2004-10-03 16:42:22.194078928 -0700
@@ -208,7 +208,7 @@ config PREEMPT
 	  or real-time system.  Say N if you are unsure.
 
 #
-# Use the generic interrupt handling code in kernel/hardirq.c:
+# Use the generic interrupt handling code in kernel/irq/:
 #
 config GENERIC_HARDIRQS
 	bool
diff -puN arch/x86_64/Kconfig~doc-remove-references-to-hardirqc arch/x86_64/Kconfig
--- 25/arch/x86_64/Kconfig~doc-remove-references-to-hardirqc	2004-10-03 16:42:22.189079688 -0700
+++ 25-akpm/arch/x86_64/Kconfig	2004-10-03 16:42:22.195078776 -0700
@@ -330,7 +330,7 @@ config X86_MCE
 endmenu
 
 #
-# Use the generic interrupt handling code in kernel/hardirq.c:
+# Use the generic interrupt handling code in kernel/irq/:
 #
 config GENERIC_HARDIRQS
 	bool
diff -puN arch/x86_64/kernel/irq.c~doc-remove-references-to-hardirqc arch/x86_64/kernel/irq.c
--- 25/arch/x86_64/kernel/irq.c~doc-remove-references-to-hardirqc	2004-10-03 16:42:22.190079536 -0700
+++ 25-akpm/arch/x86_64/kernel/irq.c	2004-10-03 16:42:22.195078776 -0700
@@ -5,7 +5,7 @@
  *
  * This file contains the lowest level x86_64-specific interrupt
  * entry and irq statistics code. All the remaining irq logic is
- * done by the generic kernel/hardirq.c code and in the
+ * done by the generic kernel/irq/ code and in the
  * x86_64-specific irq controller code. (e.g. i8259.c and
  * io_apic.c.)
  */
_
